src/components/banners/Banner.tsx

```diff
@@ -15,6 +15,15 @@ export type BannerProps = {
   message: ReactNode;
   pinned?: boolean;
   intense?: boolean;
+  // activate showMore feature
+  showMore?: boolean;
+  // text for showMore link
+  showMoreLinkText?: string;
+  // text for showless link
+  showLessLinkText?: string;
+  // additionalMessage is shown next to message when clicking showMoreLinkText
```

Comment:
We should probably type these as `ReactNode` similar to the `message` prop. We should open an issue to follow up with adding `styleOverrides` and theming to this component, but allowing `ReactNode` as the type could at least let us pass in the styling we want for now.